README.md

```diff
@@ -90,3 +101,22 @@ services:
 		arguments:
 			- .github/workflows/phpstan/exclude-return-functions-list.php
 ```
+
+#### UseTypeHintForNewMethodsRule
+
+Similarly to `UseTypedReturnForNewMethodsRule`, rule `UseTypeHintForNewMethodsRule` requires loading of an instance of `PHPStanForPrestaShop\PHPConfigurationLoader\ConfigurationLoaderInterface`,
+named `@typeHintsForNewMethodsRuleConfigurationFileLoader`. It should load an array of class methods for which
+the `UseTypeHintForNewMethodsRule` should not be applied.
+
+There is two available implementations: `PHPStanForPrestaShop\PHPConfigurationLoader\ArrayConfigurationLoader`
+and `PHPStanForPrestaShop\PHPConfigurationLoader\PHPConfigurationFileLoader`.
+
+Example with PHPConfigurationFileLoader:
+
+```neon
+services:
+	typeHintsForNewMethodsRuleConfigurationFileLoader:
+		class: PHPStanForPrestaShop\PHPConfigurationLoader\PHPConfigurationFileLoader
+		arguments:
+			- .github/workflows/phpstan/exclude-typehint-functions-list.php
```

Comment:
```suggestion
    typeHintsForNewMethodsRuleConfigurationFileLoader:
        class: PHPStanForPrestaShop\PHPConfigurationLoader\PHPConfigurationFileLoader
        arguments:
            - .github/workflows/phpstan/exclude-typehint-functions-list.php
```

use spaces :heart:

Reply:
All fixed in https://github.com/PrestaShop/phpstan-prestashop/pull/11/commits/472527f15dd4cdc443bde48ff05f62a899f2973e